configs/components/bolt.rb

```diff
@@ -3,6 +3,7 @@
 
   # Install the bolt runtime for access to rubygem-r10k:
   pkg.build_requires 'bolt-runtime'
+  pkg.build_requires 'gem-prune'
```

Comment:
Not sure this is necessary here or just in puppet-bolt.rb, not sure what the prune process would be for bolt itself versus bolt-vanagon.

Reply:
Runs without so looks like not necessary, removing.